### PR TITLE
Corrected Anomaly score calculation

### PR DESCRIPTION
**Pull request by Steven-N-Hart**
The paper calculates it this way: 
`ano_score = (1. - self.ano_para)* dis_loss + self.ano_para* res_loss`
But your original code had it calculated as 
`ano_score = (1. - self.ano_para)* res_loss + self.ano_para* dis_loss`

### CODE REVIEW

**Comment by amandaberg** [comment]
Thank you Steven!
I actually never used the ano_score calculated in the code when evaluating the method, I just used dis_loss and res_loss and combined them in another script. The paper is correct. I will update the code, thank you for pointing this out.

**Comment by Steven-N-Hart** [comment]
Also, I am looking at your Encoder loss, which is always producing NANs. I'm still digging through it with the MNIST and other datasets to find out why, but this is obviously causing the reconstructed images to not look correct. See for yourself:
```
loss = tf.reduce_mean(tf.abs(fake_images_out - fake_images_reconstructed))
loss = tfutil.autosummary('Encoder_Loss', loss)
```
